### PR TITLE
feat: dao voting view vote

### PR DESCRIPTION
**Pull request by phucledien**
**What does this PR do?**

-   [x] View proposal vote of user

**How to test**

- Use API **GET** `localhost:8200/dao-voting/proposals/vote`
- Add 2 required query
     - `discord-user-id`: Discord user ID (i.e 963123183131709480)
     - `proposal-id`: Proposal ID (i.e 1)

**Media (Loom or gif)**

CleanShot 2023-01-04 at 16 09 09@2x
CleanShot 2023-01-04 at 16 09 54@2x
CleanShot 2023-01-04 at 16 09 45@2x
CleanShot 2023-01-04 at 16 10 03@2x

### CODE REVIEW

File: pkg/routes/v1.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ func NewRoutes(r *gin.Engine, h *handler.Handler, cfg config.Config) {
 			proposalGroup.GET("/:proposal_id", h.DaoVoting.GetUserVotes)
 			voteGroup := proposalGroup.Group("/vote")
 			{
+				voteGroup.GET("", h.DaoVoting.GetVote)
```
Comment (trkhoi) [inline pkg/routes/v1.go:35]: fix giúp e cái /votes số nhiều với

**Review by trkhoi — APPROVED** [review]

File: pkg/routes/v1.go (hunk 1 above)
Comment (phucledien) [reply pkg/routes/v1.go:35]: ok ser

File: pkg/handler/dao-voting/dao-voting.go [hunk 2]
```diff
@@ -133,3 +134,39 @@ func (h *Handler) CreateProposal(c *gin.Context) {
...
 }
+
+// GetProposals     godoc
+// @Summary     Get dao votes
+// @Description Get dao votes
+// @Tags        DAO-Voting
+// @Accept      json
+// @Produce     json
+// @Param       proposal-id   query  string true  "Proposal ID"
+// @Param       user-discord-id   query  string true  "Discord ID"
```
[the remark this answers is not in the document]
Comment (phucledien) [reply pkg/handler/dao-voting/dao-voting.go:145]: ok ser

File: pkg/handler/dao-voting/dao-voting.go [hunk 3]
```diff
@@ -133,3 +134,39 @@ func (h *Handler) CreateProposal(c *gin.Context) {
...
+// @Success     200 {object} response.GetVote
+// @Router      /dao-voting/vote [get]
+func (h *Handler) GetVote(c *gin.Context) {
+	userId := c.Query("user-discord-id")
+	if userId == "" {
+		h.log.Info("[handler.GetVote] - discord id empty")
+		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errs.ErrInvalidDiscordUserID, nil))
+		return
+	}
+	proposalId := c.Query("proposal-id")
```
[the remark this answers is not in the document]
Comment (phucledien) [reply pkg/handler/dao-voting/dao-voting.go:155]: ok ser

File: pkg/handler/dao-voting/dao-voting.go [hunk 4]
```diff
@@ -133,3 +134,39 @@ func (h *Handler) CreateProposal(c *gin.Context) {
...
+// @Description Get dao votes
+// @Tags        DAO-Voting
+// @Accept      json
+// @Produce     json
+// @Param       proposal-id   query  string true  "Proposal ID"
+// @Param       user-discord-id   query  string true  "Discord ID"
+// @Success     200 {object} response.GetVote
+// @Router      /dao-voting/vote [get]
+func (h *Handler) GetVote(c *gin.Context) {
+	userId := c.Query("user-discord-id")
```
[the remark this answers is not in the document]
Comment (phucledien) [reply pkg/handler/dao-voting/dao-voting.go:149]: ok ser

**Review by anhnh12 — APPROVED** [review]

**Pull request merged**